tests/utils.go

```diff
@@ -112,7 +112,6 @@ import (
 )
 
 const (
-	KubevirtIoTest               = "kubevirt.io/test"
 	KubernetesIoHostName         = "kubernetes.io/hostname"
```

Comment:
Why do we move only a few? Where do we draw the line?

Reply:
I moved the ones that must move in order to avoid a dependency loop.
Attempting to move one small function cause the move if 1/3 of the file content.

If the others did not move, it means they are not directly dependent by the other code.
But it may be reasonable and correct to move the rest to the existing other packages or create new ones.